inst/include/Rcpp/exceptions.h

```diff
@@ -111,23 +111,51 @@ namespace Rcpp {
         throw Rcpp::exception(message.c_str());
     }                                                        // #nocov end
 
-#if defined(RCPP_USE_UNWIND_PROTECT)
     namespace internal {
 
+        inline SEXP longjumpSentinel(SEXP token) {
+            SEXP sentinel = PROTECT(Rf_allocVector(VECSXP, 1));
```

Comment:
Just curious why we're using a length-one list rather than a length-one character vector directly?

Reply:
The sentinel wraps the longjump token that is passed to `R_ContinueUnwind()`.

Reply:
Right! Sorry for the silly question.